src/utils/users.js

```diff
@@ -49,9 +49,16 @@ const getUsersInRoom = (room) => {
     return usersInRoom
 }
 
+const getRooms = () => {
+    const justRooms = users.map((user) => user.room)
+    const uniqueRooms = justRooms.filter((room, index, arr) => arr.indexOf(room) === index)
```

Comment:
```suggestion
    return [... new Set(justRooms)] // it must work if `user.room` is a string. i assume it is a string
```

Reply:
Fixed.
I saw this way online, but didn't fully understand it, so did not use.